backend/src/data/models/appointment.ts

```diff
@@ -0,0 +1,34 @@
+import { Sequelize, DataTypes, ModelCtor, Model } from 'sequelize';
+
+enum Type {
+  ONLINE = 'online',
+  OFFLINE = 'offline'
+}
+
+interface AppointmentInstance extends Model {
+  date: Date;
+  type: Type;
+  cost: number;
+  subject: string;
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+export default (orm: Sequelize): ModelCtor<AppointmentInstance> => {
+  const Appointment = orm.define<AppointmentInstance>('appointment', {
+    date: {
+      allowNull: false,
+      type: DataTypes.DATE
+    },
+    type: {
+      allowNull: false,
+      type: DataTypes.ENUM('online', 'offline')
```

Comment:
```suggestion
      type: DataTypes.ENUM(AppointmentType.OFFLINE, AppointmentType.ONLINE)
```
Do it for the all model when we have enum type